tests/unit/sftp_test.py

```diff
@@ -18,11 +20,21 @@ def sftp_session(ssh_client_session):
         del sftp_sess  # noqa: WPS420
 
 
-@pytest.fixture
-def transmit_payload():
-    """Generate a binary test payload."""
-    uuid_name = uuid.uuid4()
-    return 'Hello, {name!s}'.format(name=uuid_name).encode()
+@pytest.fixture(
+    params=(32, 1024 + 1),
+    ids=('small-payload', 'large-payload'),
+)
+def transmit_payload(request: pytest.FixtureRequest):
+    """Generate binary test payloads of assorted sizes.
+
+    The choice 32 is arbitrary small value.
+
+    The choice 1024 + 1 is meant to be 1B larger than the chunk size used in
+    sftp.pyx to make sure we excercise at least two rounds of reading/writing.
```

Comment:
It's usually a good idea to use RST syntax in docstrings.
```suggestion
    :file:`sftp.pyx` to make sure we excercise at least two rounds of reading/writing.
```